ClientApp/viking-vault/src/AttachCardModal.tsx

```diff
@@ -0,0 +1,215 @@
+import React from "react";
+import { constants } from "./ConstantVariables";
+import './AttachCardModal.css';
+
+const baseUrl = constants.baseUrl;
+var regex = /^([+-]?[1-9]\d*|0)$/;
```

Comment:
Also, please find a better name for this regex :), what do you try to match by this?